test/basedriver/helpers-e2e-specs.js

```diff
@@ -84,10 +85,9 @@ describe('app download and configuration', function () {
               return;
             }
             // for testing zip file content types
-            if (req.url.indexOf('mime-zip') !== -1) {
-              res.setHeader('content-type', 'application/zip');
-            } else if (req.url.indexOf('mime-bip') !== 1) {
-              res.setHeader('content-type', 'application/bip');
+            const contentType = new URLSearchParams(url.parse(req.url).search).get('content-type');
```

Comment:
is this doing the same as before?

Reply:
Tests may now pass a `content-type` query parameter so the test server sets the response Content-Type header accordingly. I reflected this change in existing tests.